src/debugger/debugger.ts

```diff
@@ -24,9 +31,17 @@ async function getBinaryPath(): Promise<string | undefined> {
 		const globalConfig = GlobalConfig.getInstance();
 		const kaniBinaryPath = globalConfig.getFilePath();
 
-		const playbackCommand: string = `${kaniBinaryPath} playback -Z concrete-playback --only-codegen --message-format=json`;
-		const output = execSync(`cd ${directory} && ${playbackCommand}`);
+		const playbackCommand: string = `cargo kani playback -Z concrete-playback --only-codegen --message-format=json`;
```

Comment:
I thought the extension invoked `cargo-kani` directly...

Reply:
It still does, I can clarify that with a comment but if you notice line 44, where we call the exec, we invoke cargo-kani directly. The string on this line is used to construct the arguments to be passed.